website/docs/configure_hydra/workdir.md

```diff
@@ -55,3 +55,15 @@ hydra:
         item_sep: ','
         exclude_keys: []
 ```
+
+### Customizing outputs with substitution through the CLI 
+
+Outputs can also be configured through the CLI, like any other configuration.
+
+> python train.py model.nb_layers=3 hydra.run.dir=3_layers
+
+This feature can become really powerful to write multiruns without boilerplate using substitution.
+
+> python3 train.py model.nb_layers=1,2,3,5 hydra.sweep.dir=multiruns/layers_effect hydra.sweep.subdir=\${model.nb_layers} -m
```

Comment:
```suggestion
> python train.py --multirun model.nb_layers=1,2,3,5 hydra.sweep.dir=multiruns/layers_effect hydra.sweep.subdir=\${model.nb_layers}
```

Reply:
confirm that muiltirun is not sensitive to position (should be fine).

Reply:
I had no problem with position but it is probably nicer to have it before overriding

Reply:
Yes, it's just a matter of presentation.
Also since multirun is important here, it's good to show the full flag name.